metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -95,6 +95,18 @@ object Messages {
     s"Metals is unable to start ${buildTool}. Please try to connect after starting it manually."
   )
 
+  def unknownScalafixRules(unknownRuleMessage: String): MessageParams = {
+    val regex = ".*'(.*)'.*".r
+    val rule = unknownRuleMessage match {
+      case regex(rule) => rule
+      case _ => "a rule"
+    }
```

Comment:
Can you add an example what will be matched by regex and why as a comment?

Reply:
Added an example message